web_app/calendar/app.py

```diff
@@ -63,6 +67,7 @@ def login():
 @login_required
 def logout():
     logout_user()
+    logger.info('User logged out')
```

Comment:
which user

Reply:
We cannot obtain information about user, unfortunately. Since AnonymousUserMixin object has no information about user, user id or name can't be retrieved.

Reply:
before logout_user() These is current_user.id which you can get.